packages/ui/src/common/components/statistics/StatisticBar.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { ProgressBar } from '@/common/components/Progress'
+import { Colors } from '@/common/constants'
+
+import { StatsContent, StatisticHeader, StatisticHeaderProps } from '.'
+import { FractionValue, FractionValueProps } from './FractionValue'
+
+export interface StatisticBarProps extends StatisticHeaderProps, FractionValueProps {
+  value: number
+  threshold: number
+}
+
+export const StatisticBar = ({ value, threshold, numerator, denominator, ...headerProps }: StatisticBarProps) => (
+  <div>
```

Comment:
I needed to dig into why there's a `<div>` and not the `<StaticsticBlock>` so maybe adding a  component that will tell more would be good?

I see this is meant to be used as:

```
<TwoRowStatistic>
  <div>...</div>
  <div>...</div>
</TwoRowSatistic>
```

So maybe an exported component Like `<TwoRowStatisticRow>` would be enough?

YOu could also add a generic example to: [https://deploy-preview-884--pioneer-2-storybook.netlify.app/?path=/story/common-statistics--statistic](https://deploy-preview-884--pioneer-2-storybook.netlify.app/?path=/story/common-statistics--statistic.) page on how to use them together.

Reply:
Since this kind of statistic is also used in the usual statistic layout like in [6.3.1](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Design?node-id=2014%3A595) I wanted to export a single statistic. But after adding it to the `Statistics.stories.tsx` I realize I should export a `React.Fragment` to make it reusable